pkg/estimator/client/accurate.go

```diff
@@ -67,6 +85,11 @@ func (se *SchedulerEstimator) MaxAvailableReplicas(
 	})
 }
 
+// GetPriority return the priority of this estimator
```

Comment:
// GetPriority provides the priority of this estimator.

Reply:
done